code/datums/map_elements.dm

```diff
@@ -23,6 +23,10 @@ var/list/datum/map_element/map_elements = list()
 	if(!location && objects.len)
 		location = locate(/turf) in objects
 
+	//Build powernets
+	for(var/obj/structure/cable/C in objects)
+		C.rebuild_from()
```

Comment:
Yes, but once a cable gets a power net built, all adjacent cables do too so calling the proc is a waste of cpu time of the already have a power net.